shumai/tensor/op_to_flops.ts

```diff
@@ -0,0 +1,8 @@
+export function opToFlops(op: string, args: any[]): bigint {
+  switch (op) {
+    case 'matmul':
+      return BigInt(Math.round(Math.pow(args[0].elements, 2.37188)))
```

Comment:
Doesn't account for scalars but wasn't sure if there was a more accurate calculation.

Reply:
you'll want to do something like `args[0].shape[0] * args[1].shape[0] * args[1].shape[1] * 2`, but this is fine to check in for now